plugins/lookup/onepassword_raw.py

```diff
@@ -45,6 +45,12 @@
           - Only works with 1Password CLI version 2 or later.
         type: string
         version_added: 7.1.0
+      connect_host:
+        description: The host for 1Password Connect. Must be used in combination with C(connect_token).
+        type: str
```

Comment:
```suggestion
        type: str
        version_added: 7.4.0
```
Similar for all other new plugin options.